packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -683,6 +683,12 @@ const main = async (opts) => {
         })
     }
 
+    if (context.answers.project.commerce.instanceUrl) {
+        // Remove protocol since we only use this to setup the OCAPI proxy
+        const url = new URL(context.answers.project.commerce.instanceUrl)
+        context.answers.project.commerce.instanceUrl = url.hostname
```

Comment:
NIT: Do we want to store the hostname in a separate variable to make sure we don't lose the original answer just in case ?

Reply:
I went with this implementation for 2 reasons:

1. The only place where we use the instanceUrl is when setting up the OCAPI proxy (outside of the OCAPI proxy it's not used anywhere)
2. The OCAPI proxy in defaults.js does not need the protocol. However, the validation we have in the generator takes the string passed in by the user and puts it into a new Url object, which requires an absolute url